test/cypress/fixtures/cardsProgress.json

```diff
@@ -0,0 +1,134 @@
+[
+  {
+    "title": "Týmová pravidelnost",
+    "icon": "ion-bicycle",
+    "progress": 80,
+    "duration": {
+      "current": 24,
+      "total": 30
+    },
+    "prizes": [
+      {
+        "icon": "",
+        "placement": 1,
+        "label": "místo"
+      }
+    ],
+    "stats": [
+      {
+        "title": "Váš podíl na výsledku",
+        "items": [
+          {
+            "id": "1",
+            "text": "80% pravidelnost"
+          }
+        ]
+      },
+      {
+        "title": "Postavení v žebříčku",
+        "items": [
+          {
+            "id": "1",
+            "text": "2. Automati a 1 další (80 %)"
+          },
+          {
+            "id": "2",
+            "text": "3. Váš tým a 2 další (60 %)"
+          },
+          {
+            "id": "3",
+            "text": "4. Tygři a 2 další (40 %)"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "title": "Vaše pravidelnost",
+    "icon": "ion-bicycle",
+    "progress": 60,
+    "duration": {
+      "current": 24,
+      "total": 30
+    },
+    "prizes": [
+      {
+        "icon": "",
+        "placement": 3,
+        "label": "místo"
+      }
+    ],
+    "stats": [
+      {
+        "title": "Váš podíl na výsledku",
+        "items": [
+          {
+            "id": "1",
+            "text": "80% pravidelnost"
+          }
+        ]
+      },
+      {
+        "title": "Postavení v žebříčku",
+        "items": [
+          {
+            "id": "1",
+            "text": "2. Automati a 1 další (80 %)"
+          },
+          {
+            "id": "2",
+            "text": "3. Váš tým a 2 další (60 %)"
+          },
+          {
+            "id": "3",
+            "text": "4. Tygři a 2 další (40 %)"
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "title": "Vaše zelené kilometry",
+    "icon": "ion-bicycle",
+    "progress": 70,
+    "duration": {
+      "current": 24,
+      "total": 30
+    },
+    "prizes": [
+      {
+        "icon": "",
+        "placement": 5,
+        "label": "místo"
+      }
+    ],
+    "stats": [
+      {
+        "title": "Váš podíl na výsledku",
+        "items": [
+          {
+            "id": "1",
+            "text": "80% pravidelnost"
```

Comment:
Same case https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/539#discussion_r1740461555.

Reply:
Ok. I would rather fix this in a separate PR. I did a refactor of mocks here, but the changes to fix this include other components, such as `CardProgress` and `CardProgressSlider`.

Reply:
@tmszi if you agree, I would merge this PR and continue in a new one with refactor for `CardProgress` data.

Reply:
Ok I agree.